smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs

```diff
@@ -58,7 +58,11 @@ pub fn refund_bank_msg(
 
 /// swap will always swap over the CL pool. In the future we may expand the
 /// feature such that it chooses best swaps over all routes
-pub fn swap_msg(deps: &DepsMut, env: &Env, params: SwapParams) -> Result<CosmosMsg, ContractError> {
+pub fn swap_msg(
+    deps: &DepsMut,
+    sender: String,
```

Comment:
Changing the type to `Addr` improves type safety

Reply:
done